README.md

```diff
@@ -1,23 +1,350 @@
-<p align="center">
-  <a href="https://grafbase.com">
-    <img alt="Grafbase logo" src="https://github.com/grafbase/grafbase/assets/14347895/9580d0f7-d50f-4d30-8dd0-dcea1a83409e" width="1000">
-    <h3 align="center">Grafbase</h3>
-  </a>
-</p>
-
-<p align="center">
-  The GraphQL Federation platform
-</p>
-
-<p align="center">
-  <a href="/examples"><strong>Examples</strong></a> ·
-  <a href="https://grafbase.com/docs"><strong>Docs</strong></a> ·
-  <a href="https://grafbase.com/cli"><strong>CLI</strong></a> ·
-  <a href="https://grafbase.com/docs/self-hosted-gateway"><strong>Gateway</strong></a> ·
-  <a href="https://grafbase.com/community"><strong>Community</stong></a> ·
-  <a href="https://grafbase.com/changelog"><strong>Changelog</strong></a>
-</p>
-
-<p align="center">
-  Compose all your data sources into a unified graph. Move faster without breaking things using advanced governance and analytics features.
-</p>
+# **![Grafbase](https://github.com/grafbase/grafbase/assets/14347895/9580d0f7-d50f-4d30-8dd0-dcea1a83409e)** **Grafbase**
+
+**The High-Performance GraphQL Federation Platform for Mission-Critical APIs**
+
+[Website](https://grafbase.com) • [Documentation](https://grafbase.com/docs) • [CLI](https://grafbase.com/cli) • [Community](https://grafbase.com/community) • [Blog](https://grafbase.com/blog)
+
+## **What is Grafbase?**
+
+Grafbase is a self-hosted, Rust-powered GraphQL Federation Gateway designed for high-scale, mission-critical applications. Whether you're unifying microservices, legacy systems, or third-party APIs—Grafbase helps teams ship faster and more safely.
+
+### **Built for GraphQL Federation 2.0**
```

Comment:
What is GraphQL Federation 2.0? need a better title

Reply:
Updated but open to ideas on replacing the title